src/app/features/home/post-capture-tab/series/series.page.ts

```diff
@@ -1,14 +1,15 @@
-import { Component, OnInit } from '@angular/core';
-import { ActivatedRoute, Router } from '@angular/router';
+import { Component } from '@angular/core';
+import { ActivatedRoute } from '@angular/router';
 @Component({
   selector: 'app-series',
   templateUrl: './series.page.html',
   styleUrls: ['./series.page.scss'],
 })
-export class SeriesPage implements OnInit {
-  id: string | null | undefined;
-  cover: string | null | undefined;
-  collectionGeneral = [
+export class SeriesPage {
+  readonly id = this.route.snapshot.paramMap.get('id');
```

Comment:
We do not need `snapshot` to get `paramMap`, use `this.route.paramMap` directly. See the official docs for the details.